turbo.json

```diff
@@ -30,7 +30,11 @@
     "ARBITRUM_ALCHEMY_API_KEY",
     "HARDHAT_CHAIN_ID",
     "HARDHAT_MNEMONIC",
+    "CYPRESS_COMPONENT_TEST",
     "CYPRESS_DAPP",
+    "CYPRESS_TENDERLY_ACCOUNT_SLUG",
+    "CYPRESS_TENDERLY_PROJECT_SLUG",
+    "CYPRESS_TENDERLY_ACCESS_KEY",
```

Comment:
Please add these to [the workflow](https://github.com/curvefi/curve-frontend/blob/e6ea3446ab98287323f38533f1811529530a477f/.github/workflows/ci.yaml#L29) and send an admin the actual secrets